source/content/guides/autopilot/07-troubleshoot.md

```diff
@@ -373,11 +373,11 @@ Site is running a framework that is not supported. Supported frameworks are:
 
 ### Diagnosis
 
-A [WordPress Composer Managed](/guides/wordpress-composer/pre-ga/wordpress-composer-managed) site which is not using Composer is not supported by Autopilot.
+A [WordPress Composer Managed](/guides/wordpress-composer/wordpress-composer-managed) site which is not using Composer is not supported by Autopilot.
```

Comment:
I know you were just updating URLs, but can this sentence be rewritten to make sense?

Reply:
That document is owned by @pantheon-systems/lifecycle-ops (Autopilot). I am not sure the context of the note that was added.

Reply:
AFAIK What it means is Build Tools sites do not work with Autopilot. If you are using Composer, you must use IC.

Reply:
```suggestion
A Composer-based WordPress site using [Build Tools](https://docs.pantheon.io/guides/build-tools/) is not supported by Autopilot.
```

@pwtyler this look good?